### PR TITLE
PDP-1499: PM recommended README fixes

### PR DESCRIPTION
**Pull request by nashtapu-tibco**
https://docs.google.com/document/d/1VOaVQWsE_u9XY6G1HWjIevWdRDwaNjoEidtFdPtEF4U/edit

### CODE REVIEW

File: docs/dp-cluster-workshop/eks/README.md [hunk 1]
```diff
@@ -312,7 +312,7 @@ nginx   k8s.io/ingress-nginx   <none>       7h11m
 The `nginx` ingress class is the main ingress that DP will use. The `alb` ingress class is used by AWS ALB ingress controller.
 
 > [!IMPORTANT]
-> You will need to provide this ingress class name i.e. nginx to TIBCO Control Plane when you deploy capability.
+> You will need to provide this ingress class name i.e. nginx to TIBCO® Control Plane when you deploy capability.
 
 ### Storage Class
```
Comment (suyoggandhi-tibco) [inline docs/dp-cluster-workshop/eks/README.md:317]: Should we add the Storage Class step after Setup EFS? EFS ID created as part of Setup EFS is getting used in Storage Class step.

File: docs/dp-cluster-workshop/eks/README.md [hunk 2]
```diff
@@ -221,19 +221,19 @@ In this section, we will install ingress controller and storage class. We have m
...
+* [Amazon Route 53](https://aws.amazon.com/route53/): to manage DNS. You can register your Data Plane domain in Route 53. And give permission to external-dns to add new record.
 * [AWS Certificate Manager (ACM)](https://docs.aws.amazon.com/acm/latest/userguide/acm-overview.html): to manage SSL certificate. You can create a wildcard certificate for `*.<DP_DOMAIN>` in ACM.
 * aws-load-balancer-controller: to create AWS ALB. It will automatically create AWS ALB and add SSL certificate to ALB.
 * external-dns: to create DNS record in Route 53. It will automatically create DNS record for ingress objects.
 
-For this workshop work; you will need to 
+For this workshop, you will need to
 * register a domain name in Route 53. You can follow this [link](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/domain-register.html) to register a domain name in Route 53.
 * create a wildcard certificate in ACM. You can follow this [link](https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-request-public.html) to create a wildcard certificate in ACM.
```
Comment (suyoggandhi-tibco) [inline docs/dp-cluster-workshop/eks/README.md:239]: Adding a comment here as not able to comment on the Setup EFS and Ingress Controller section.

Let's add the below line at the end of the Setup EFS section to make more sense. Currently, it's in the Ingress Controller section 

'After running above script; we will get an EFS ID output like `fs-0ec1c745c10d523f6`. We will need to use this value to deploy `dp-config-aws` helm chart.'

**Review by suyoggandhi-tibco — APPROVED** [review]
Added minor comments.

File: docs/dp-cluster-workshop/eks/README.md (hunk 2 above)
Comment (nashtapu-tibco) [reply docs/dp-cluster-workshop/eks/README.md:239]: Addressed.

File: docs/dp-cluster-workshop/eks/README.md (hunk 1 above)
Comment (nashtapu-tibco) [reply docs/dp-cluster-workshop/eks/README.md]: Addressed.

**Review by suyoggandhi-tibco — APPROVED** [review]

**Pull request merged**